test/foundry/MarketFactory/Unit.t.sol

```diff
@@ -308,8 +307,8 @@ contract MarketFactoryUnitTest is Test {
         uint256 currentNonce = factory.nonce();
         bytes32 salt = keccak256(abi.encode(block.chainid, currentNonce));
         address expectedOrderTokenAddress = orderTokenDeployer.computeTokenAddress(salt);
-        uint64 priceBookDeployerNonce = vm.getNonce(address(priceBookDeployer));
-        address expectedPriceBookAddress = Create1.computeAddress(address(priceBookDeployer), priceBookDeployerNonce);
+        address expectedPriceBookAddress = priceBookDeployer.computeArithmeticPriceBookAddress(a, d);
+        assert(factory.deployedGeometricPriceBook(a, d) == address(0));
```

Comment:
It would be better to check this function after deployment is finished.

Reply:
https://github.com/clober-dex/core/pull/40/commits/8a4f9b2aec61198409c50dc8b50e658e2e154276 @JhChoy